pyproject.toml

```diff
@@ -77,6 +77,7 @@ full = [
     "networkx",
     "distinctipy",
     "matplotlib",
+    "huggingface_hub",
```

Comment:
I am not sure that this huggingface_hub should be in full.

Reply:
Where do you think is best?
So I moved it here @alejoe91 did not want it `dependencies`

That said, this can wait, so I am removing it from here and adding it on the SingleChannelDenoiser test.